docs/using.rst

```diff
@@ -205,6 +205,26 @@ perform the authentication procedure and/or clean up after it by using
 the ``--manual-auth-hook`` and ``--manual-cleanup-hook`` flags. This is
 described in more depth in the hooks_ section.
 
+.. _combination:
+
+Combining plugins
+-----------------
+
+Sometimes you may want to specify a combination of distinct authenticator and
+installer plugins. To do so, specify the authenticator plugin with
+``--authenticator`` or ``-a`` and the installer plugin with ``--installer`` or
+``-i``.
+
+For instance, you may want to create a certificate using the webroot_ plugin
+for authentication and the apache_ plugin for installation, perhaps because you
+use a proxy or CDN for SSL and only want to secure the connection between them
+and your origin server, which cannot use the tls-sni-01_ challenge due to the
+intermediate proxy.)
```

Comment:
Sorry about that. I tried very hard to make this two sentences but everything I came up with was more annoying and repetitive th